tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/BrokerRequestTests.cs

```diff
@@ -184,6 +184,7 @@ public async Task BrokerGetAccountsTestAsync()
             var mockBroker = Substitute.For<IBroker>();
             var expectedAccount = Substitute.For<IAccount>();
             mockBroker.GetAccountsAsync(TestConstants.ClientId).Returns(new[] { expectedAccount });
+            mockBroker.CanInvokeBroker().Returns(true);
```

Comment:
do you have a test where .CanInvokeBroker().Returns(false) and ensures the other route is taken?

Reply:
good point, I added that